### ISSUE TITLE
crudbooster.com ? Future of your amazing dev ?

### ISSUE THREAD

**Issue by Benoit75** [crocodic-studio/crudbooster#1449]
Hello @fherryfherry,
Thank's for your work. I would like to start a work related project with CB and 
I'm a little concern about the futur of your package. 

Your site crudbooster.com is out of order and i can't find any news about a crudbooster 's version compatible whith the current version of Laravel (5.8)

Is it possible to have some news ? 
Many thank's
Regard.
B.

**Comment by labschool** [comment]
@fherryfherry Your work is excellent. We hope soon to have your new version.

**Comment by geoffsari** [comment]
Yes, totally agree with the other user. Very excellent work overall. Hope you find a way to continue supporting this project.

**Comment by lobointerestelar** [comment]
You have an amazing Crud Generator, even much better than many big companies' solutions. 
Your need to find a way to offer a support solution, even paid. I was trying to contact you via crudbooster@gmail.com. But I didn't get any answer from you. Congratulation for this amazing tool.
